modules/gdscript/tests/scripts/analyzer/errors/getter_no_return.gd

```diff
@@ -0,0 +1,6 @@
+var y:
+	get:
+		pass
+
+func test():
+	print("Not all code paths in the getter function return a value.")
```

Comment:
```suggestion
	pass
```

It seems to me that duplicating the error text here is confusing. The text is not even printed.